contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeEditListItem.vue

```diff
@@ -1,80 +1,99 @@
+                    drag_indicator
+                  </Icon>
+                </VBtn>
+              </transition>
+            </VListTileAction>
+            <VListTileAction class="mx-2 select-col" @click.stop>
+              <Checkbox
+                v-model="selected"
+                :disabled="copying"
+                class="mt-0 pt-0"
+                @dblclick.stop
+              />
+            </VListTileAction>
+          </template>
 
-        <template #actions-end>
-          <VListTileAction :aria-hidden="!active" class="action-icon px-1">
-            <Menu v-model="activated">
-              <template #activator="{ on }">
+          <template #actions-end>
+            <VListTileAction class="action-icon px-1" @click.stop>
+              <transition name="fade">
                 <IconButton
-                  icon="optionsVertical"
-                  :text="$tr('optionsTooltip')"
+                  icon="edit"
                   size="small"
+                  :text="$tr('editTooltip')"
                   :disabled="copying"
-                  v-on="on"
                   @click.stop
+                  @click="showTitleDescriptionModal"
                 />
-              </template>
-              <ContentNodeOptions v-if="!copying" :nodeId="nodeId" />
-            </Menu>
-          </VListTileAction>
-        </template>
+              </transition>
+            </VListTileAction>
+            <VListTileAction :aria-hidden="!active" class="action-icon px-1">
+              <Menu v-model="activated">
+                <template #activator="{ on }">
+                  <IconButton
+                    icon="optionsVertical"
+                    :text="$tr('optionsTooltip')"
+                    size="small"
+                    :disabled="copying"
+                    v-on="on"
+                    @click.stop
+                  />
+                </template>
+                <ContentNodeOptions v-if="!copying" :nodeId="nodeId" />
+              </Menu>
+            </VListTileAction>
+          </template>
 
-        <template #context-menu="{ showContextMenu, positionX, positionY }">
-          <ContentNodeContextMenu
-            :show="showContextMenu"
-            :positionX="positionX"
-            :positionY="positionY"
-            :nodeId="nodeId"
-          />
-        </template>
-      </ContentNodeListItem>
-    </template>
-  </DraggableItem>
+          <template #context-menu="{ showContextMenu, positionX, positionY }">
+            <ContentNodeContextMenu
+              :show="showContextMenu"
+              :positionX="positionX"
+              :positionY="positionY"
+              :nodeId="nodeId"
+            />
+          </template>
+        </ContentNodeListItem>
+      </template>
+    </DraggableItem>
+    <EditTitleDescriptionModal
+      v-if="showQuickEdit.titleDescription"
+      :node="contentNode"
+      @close="showQuickEdit.titleDescription = false"
+    />
```

Comment:
Since this modal isn't specific to individual content nodes, or `ContentNodeEditListItem` component itself, it seems slightly better to have it implemented outside of this component. As a modal, there should only ever be one of them open at a time. Having it implemented singularly outside of `ContentNodeEditListItem` then better aligns its implementation with its behavior. You could have this component emit events upwards that can be handled where we render multiple of these as a group.

We have the same issue with `ContentNodeContextMenu` but we've seen how that can increase memory usage and have come to recognize that its a pattern we should correct.